config/locales/jobs.yml

```diff
@@ -46,10 +46,12 @@ en:
     no_jobs:
       heading: Try another search
       intro: 'You may get more results if you:'
-      radius_suggestions:
+      wider_search_suggestions:
         heading:
           keyword_html: Try browsing for <span class='text-wrap-apostrophe'>%{keyword}</span> in a wider location search
           no_keyword: Try browsing in a wider location search
+        suggestion_html: <li class="govuk-link wider-radius-gtm">%{search_link} %{count}</li>
```

Comment:
We should be passing in `govuk_link_to` helper, not writing html in the translation